ops/scripts/create_dev_sandbox.sh

```diff
@@ -112,10 +112,14 @@ sed -i '' '/          - development/ {a\
           - '"$1"'
 }' .github/workflows/reset-db.yaml
 
-sed -i '' '/          - development/ {a\
+sed -i '' '/          - backup/ {a\
           - '"$1"'
 }' .github/workflows/migrate.yaml
 
+sed -i '' '/          - development/ {a\
+          - '"$1"'
+}' .github/workflows/deploy-manual.yaml
```

Comment:
@erinysong adding this to the create makes sense but you then have to make sure it gets removed in the destroy yaml